design/specs/navigation.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Navigation Updates
+description: 
+published: true
+date: 2022-08-25 18:02:53
+tags: 
+editor: markdown
+dateCreated: 2022-08-25 18:02:58
+---
+
+## Context
+
+Mathesar users must be able to navigate between pages and know where they are at any point while using the system.
+
+### Design Goals
+
+- Users clearly understand which page they're on
+- Users easily navigate to other pages
+- Users may anticipate the result of navigation interactions
+
+## Navigation structure diagram
+
+The navigation design will be based on the following structure:
+
+| 1st Level         | 2nd Level       | 3rd Level     | 4th Level   |
+|-------------------|-----------------|---------------|-------------|
+| Database Page     | Schema Page     | Table Page    | Record Page |
+|                   | Data Explorer   | Query Page    |             |
+|                   | Data Import     |               |             |
+
+- 1st level: Database
+- 2nd level: Schema
+- 3rd level: Entities
+- 4th level: Record
+
+[Navigation by Page](https://balsamiq.cloud/sjfrgln/pbe3tnu/r7FA3?f=N4IgUiBcCMA0IDkpxAYWfAMhkAhHAsjgFo4DSUA2gLoC%2BQA%3D)
+
+## Phasing Out Tab Navigation
+
+Tabs will be phased out as part of the new navigation improvements.
+
+## Navigation Methods
+
+Navigation within Mathesar will take place in three different ways:
+
+- Through the site header
+- Through links in pages
+- Direct access through page URLs
+
+### Navigation via site header
+
+- The logo in the site header will lead visitors to the site's root, in this case the database homepage.
+- A breadcrumb-like navigation will be displayed at the top part of the site's header as the user navigates through the pages.
+- When users click on a breadcrumb element, a [page-selector](#page-selector) component is displayed with a list of schemas or entities based on the breadcrumb level.
+- Upon reaching the record level, the top navigation will show a 'Go to Record' action, which will open the [Record Selector](/engineering/specs/record-selector) component.
+- The current page will not be included in the breadcrumb path.
```

Comment:
I'm not sure what this means. Could you maybe include examples of breadcrumbs on each type of page?

Reply:
I've included examples for each type of page.

Reply:
Looks good, thanks.